src/front/ticketer/lib/main.dart

```diff
@@ -1,7 +1,10 @@
 import 'package:ticketer/auth/widget_tree.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+
+void main() async {
+  await dotenv.load(fileName: ".env");
```

Comment:
I think that in the past I had some problems with dotenv file being called “.env” and had to change it to “dotenv” it might be an issue on Android or iOS. Will have to look into it.